concent_api/core/validation.py

```diff
@@ -106,6 +106,23 @@ def validate_list_of_identical_task_to_compute(list_of_task_to_compute: List[mes
     return True
 
 
+def validate_list_of_identical_report_computed_task(list_of_report_computed_task: List[message.tasks.ReportComputedTask]):
+    assert isinstance(list_of_report_computed_task, list)
+    assert all([isinstance(report_computed_task, message.tasks.ReportComputedTask) for report_computed_task in list_of_report_computed_task])
+
+    if len(list_of_report_computed_task) <= 1:
+        return False
```

Comment:
Why aren't all messages on a list containing one message considered identical? Same for zero messages. I think `True` would make more sense here.

Reply:
@cameel Function `validate_list_of_identical_task_to_compute` resolves most of this issues, that's why I have suggested to use it and making it generic.

Reply:
OK. It's fine. But please make sure that issues that are not resolved there are addressed in this pull request.